Fitness/Views/ProBioPageViewController.swift

```diff
@@ -0,0 +1,309 @@
+//
+//  ProBioPageViewController.swift
+//  
+//
+//  Created by Cameron Hamidi on 2/28/19.
+//
+
+import UIKit
+
+class ProBioPageViewController: UIViewController, UICollectionViewDataSource, UICollectionViewDelegate, UICollectionViewDelegateFlowLayout {
```

Comment:
we should conform to each of these protocols in its own extension.
i.e.
```
extension ProBioPageViewController: UICollectionViewDataSource {
...
}

extension ProBioPageViewController: UICollectionViewDelegate {
...
}
...
```

Reply:
Will do